src/packages/end-to-end/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
```

Comment:
Why is all this commented out?

Reply:
thats the default config after `npx tsc --init`